packages/frontend/src/features/videos/page/MovieListView.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import {useRef} from 'react';
+import styled from 'styled-components';
+import {useSelector,RootStateOrAny} from "react-redux"
+
+import VideoItem from '../components/movieList/VideoItem';
+import BannerContainer from '../../common/components/layout/banner/BannerContainer';
+import NavBarContainer from '../../common/components/layout/navBar/NavBarContainer';
+import Filters from '../../filters/components/Filters';
+import {Device} from '../../common/styles/constans/Device';
+import {useFillerItems} from '../hooks/useFillerItems';
+
+const Wrapper = styled.div`
+display:flex;
+flex-direction:row;
+padding-top:110px;
```

Comment:
jesteś pewien, ze nie da sie tego inaczej ustawić jak padding 110 px ?

Reply:
co to za niewielki padding :)

Reply:
Header ma position fixed i jak ten padding jest mniejszy to kafelki z filmami są przez niego zakrywane.

Reply:
To wysokosc headera do stalych i do użycia tu żeby się to nie rozjechało przy zmianie wysokosci hedera